src/pkg/token/token_test.go

```diff
@@ -92,6 +92,42 @@ func TestRaw(t *testing.T) {
 	assert.NotNil(t, rawTk)
 }
 
+func TestRawWithClockSkew(t *testing.T) {
```

Comment:
IMO, it might be make more sense to write the `TestNewWithClockSkew` instead of `TestRawWithClockSkew` since the we do validation directly on the `New()`

Reply:
Fixed by a788b68